impl/src/test/java/org/jboss/arquillian/warp/impl/testutils/TestJavassistAndSeparatedClassLoader.java

```diff
@@ -39,8 +39,10 @@ public void test() throws Exception {
 
         JavaArchive archive = ShrinkWrap.create(JavaArchive.class).add(new CtClassAsset(ctClass));
 
-        ClassLoader classLoader = new ShrinkWrapClassLoader(ClassLoaderUtils.getBootstrapClassLoader(), archive);
+        ShrinkWrapClassLoader classLoader = new ShrinkWrapClassLoader(ClassLoaderUtils.getBootstrapClassLoader(), archive);
```

Comment:
Use `try-with-resources` since ShrinkWrapClassLoader implements [AutoCloseable](https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html).

Reply:
Done. Just learned something ;-)

Reply:
Cool! :)